zulip/zulip/__init__.py

```diff
@@ -1208,6 +1208,19 @@ def get_stream_topics(self, stream_id):
             method='GET'
         )
 
+    def get_user_groups(self, request=None):
```

Comment:
What is `request` for? The original endpoint doesn't accept any parameters, right?

Reply:
@YagoGG ,you are right, removed `request`.